### ISSUE TITLE
Instructions mentions Linux release, but no Linux or OSX release is present

### ISSUE THREAD

**Issue by Sweeper5** [CiarenceW/G36C_plugin#2]
The downloads for the G36C here seems to be Windows only, but the instructions mentions it existing for Linux.
However no Linux version is provided, may need a delete of the OSX and Linux instructions if this is an outlier and only for Windows setups.

**Comment by CiarenceW** [comment]
yeah it used to support but then I got lazy and stopped building for Linux and Mac. lol

**Comment by Sweeper5** [comment]
Alright, well that explains the problems I have been encountering then.
Well the AR-15 was a bust, but the MP5 have been the most enjoyable to use so far and that one still works.
Not sure I want to open another can of worms as modding under proton is though, was quite problematic with Derail Valley.

Thank you for what you made at least, you did make me pick up the game as I really wanted to toy with some SMGs and rifles.

**Comment by CiarenceW** [comment]
what do you meannnnnn the AR-15 was a bust, also, you win, I'll update all the guns, someday, when I get Unity working again :(

**Comment by Sweeper5** [comment]
Well I guess the California compliant model or bolt action 223 didn't really appeal to me as much, but I know you wanted something unique.
Don't stress out with getting this sorted, I mean I will gladly use it and play with what you make, but I decided to help you out by pointing out the minor issues I could find, maybe avoid a few issues on your tracker in the future from others that stumble on it to have a little gun play / porn in Receiver :D
